### DISCUSSION TITLE
Additional custom properties for FluentTreeItem

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#1164] [Q&A] [answered]
Hi, is it possible to add additional custom properties to a FluentTreeItem and access those in OnExpandedChange, OnSelectedChange and other events raised by the tree. Thanks.

**Comment by vnbaaij** [comment, marked as answer]
Every `FluentComponentBase` derived component has the `Data` parameter (of type Object). You can use that to store and access any kind of extra information.
